cypress_shared/components/bedspaceSearchResult.ts

```diff
@@ -36,6 +36,8 @@ export default class BedspaceSearchResult extends Component {
           cy.get('ul[data-cy-bedspace-key-characteristics] > li').should('contain', characteristic.name)
         })
 
+        cy.get('div[data-cy-premises-notes]').should('contain', this.result.premises.notes)
```

Comment:
```suggestion
        cy.get('p[data-cy-premises-notes]').should('contain', this.result.premises.notes)
```
Only if you think `p` would be better to use